test/runtests.jl

```diff
@@ -2,6 +2,7 @@
 using JumpProcesses, DiffEqBase, SafeTestsets
 
 @time begin
+    @time @safetestset "Bracketing Tests" begin include("bracketing.jl") end
```

Comment:
I'd put these lower down the list. Maybe by the composition-rejection table tests.

Reply:
Done. My logic was that unit tests should go first becaues they are faster than integration tests.

Reply:
That makes sense, but we don't currently separate them (perhaps we should), so I'd prefer to keep the tests ordered by how "fundamental" the component is in some sense. If we want to consider restructuring them in the future though that would be a good consideration!